### PR TITLE
CC-3572 Fix transfer noWait property type

### PR DESCRIPTION
**Pull request by m7moud**
- Developer(s): @m7moud

- Ticket: https://spryker.atlassian.net/browse/CC-3572

- Academy PR: -

- Release Group: https://release.spryker.com/release-groups/view/1486

#### Please confirm

- [X] No new OS components - or they have been approved by legal department

#### Documentation

- [X] Functional documentation provided or in progress?
- [X] Integration guide for projects provided or not needed?
- [X] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   RabbitMq              | minor                 |                       |

#### Release Notes

{skip}

-----------------------------------------

##### Change log

Improvements

- Adjusted `RabbitMqOptionTransfer` object property `noWait` type to boolean.

### CODE REVIEW

**Comment by dereuromark** [comment]
I am fine with the minor
Some tests for both before and after would have been nice for BC reasons.

**Comment by m7moud** [comment]
@dereuromark Can you clarify how can I write some tests for before and after, I can test the current (after), but how can I test before?

**Comment by dereuromark** [comment]
Well, if there were some tests already they would still pass. You could also make a test fixture (simulated transfer) with old way, but we would be fine for now with only "current" way tested IMO.

File: src/Spryker/Client/RabbitMq/Model/Helper/QueueEstablishmentHelper.php [hunk 1]
```diff
@@ -64,6 +64,9 @@ public function createExchange(AMQPChannel $channel, RabbitMqOptionTransfer $que
      */
     protected function convertTransferToArray(RabbitMqOptionTransfer $queueOptionTransfer)
     {
-        return $queueOptionTransfer->toArray();
+        $queueParams = $queueOptionTransfer->toArray();
+        $queueParams['no_wait'] = filter_var($queueParams['no_wait'], FILTER_VALIDATE_BOOLEAN); // Just a precaution for BC
```
Comment (ehsanmx) [inline src/Spryker/Client/RabbitMq/Model/Helper/QueueEstablishmentHelper.php]: use constant 'no_wait'

**Pull request merged**